### ISSUE TITLE
Need a way to pass in SSL configuration

### ISSUE THREAD

**Issue by bdeboe** [caretdev/llama-iris#2]
The current entry points seem to only accept a connection string, or the composing hostname/port/.... 

In order to connect over a secure connection, we need a way to pass the SSL configuration settings to the SQLAlchemy `create_engine()` call.

**Comment by daimor** [comment]
```
sslcontext = ssl.create_default_context(cafile="certificateSQLaaS.pem")

vector_store = IRISVectorStore.from_params(
    connection_string=CONNECTION_STRING,
    table_name="demo",
    engine_args={"connect_args": {"sslcontext": sslcontext}},
)
```

**Comment by bdeboe** [comment]
addressed per d228b28
